### PR TITLE
Add search feature

### PR DESCRIPTION
**Pull request by kepet19**

### CODE REVIEW

File: components/util.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export function getAgeFromTime(birthDayTime) {
+  let timeDiff = Date.now() - birthDayTime.getTime();
+
+  // I really hope not this is going to last forever!
+  // One magic number magics numbers everywhere!!!
+  return new Date(timeDiff).getFullYear() - 1970;
```
Comment (Joergenson) [inline components/util.js:6]: you know. Date objects in javascript has getFullYearUTC :3

**Review by Joergenson — CHANGES_REQUESTED** [review]

File: components/util.js (hunk 1 above)
Comment (Joergenson) [reply components/util.js:6]: same with month and day

Comment (kepet19) [reply components/util.js:6]: Date object backed by unix time. 
unix time only back to 1970. 
that why I have to minues it. 
even if I use getUTCFullYear() I would still have to do it. 
getFullYear is only the localtime where UTC is the universial time. :b

Comment (kepet19) [reply components/util.js:6]: You want UTC or local time ?

**Review by SnaBe — APPROVED** [review]
Looks good, as for dates, [moment.js](https://momentjs.com/) provides easy parsing and display options, if need be. 😄

File: components/util.js (hunk 1 above)
Comment (SnaBe) [reply components/util.js:6]: Can't we use the local time of the phone?

**Comment by kepet19** [comment]
> Looks good, as for dates, [moment.js](https://momentjs.com/) provides easy parsing and display options, if need be. smile

It seems like a really nice Library. 
if we need better time support we can change to it? in a later pr :b

File: components/util.js (hunk 1 above)
Comment (kepet19) [reply components/util.js:6]: Let me know If I should change anything  @Joergenson ?

Comment (kepet19) [reply components/util.js:6]: On my machine 
```js
console.log(new Date().getUTCHours());
console.log(new Date().getHours());
```
returns the same thing. 
I thought getHours() returned localtime and not UTC.
We can change to another Library, but I will argue 24 hours of at max doesn't matter in our Application.

**Review by Joergenson — APPROVED** [review]

**Pull request merged**